docs/cloud/rahti/storage/objectstorage.md

```diff
@@ -0,0 +1,280 @@
+# Allas storage in Rahti
+
+More information about [Allas](../../../data/Allas/index.md)  
+
+## Backup to Allas
+
+There are different ways to backup to Allas from Rahti. We will show you two examples:  
+  - The first one is using another pod to copy the content of your persistent volume to Allas.  
+  - The second one is a bash script that you have to execute from your local machine.
+
+## First example: using another pod
+
+For this first example, we assume that we have a `nginx` deployment running with a `PersistentVolumeClaim`. We provide the files for testing purposes.
+
+First, we build a nginx image with this Dockerfile: (since it's not possible to use the regular `nginx` image in OpenShift)  
+
+```Dockerfile
+FROM nginx:stable
+
+ENV LISTEN_PORT=8080
+
+# support running as arbitrary user which belogs to the root group
+RUN chmod g+rwx /var/cache/nginx /var/run /var/log/nginx
+
+# users are not allowed to listen on privileged ports
+RUN sed -i.bak "s/listen\(.*\)80;/listen ${LISTEN_PORT};/" /etc/nginx/conf.d/default.conf
+
+# comment user directive as master process is run as user in OpenShift anyhow
+RUN sed -i.bak 's/^user/#user/' /etc/nginx/nginx.conf
+
+EXPOSE 8080
+```
+
+Then you can deploy this `nginx` with this Deployment file:   
+
+```yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: nginx
+  labels:
+    name: nginx
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: nginx
+  template:
+    metadata:
+      labels:
+        app: nginx
+    spec:
+      containers:
+      - name: nginx
+        image: <your_custom_nginx_image>
+        resources:
+          limits:
+            memory: "128Mi"
+            cpu: "500m"
+        ports:
+          - containerPort: 8080
+        volumeMounts:
+        - name: myvol
+          mountPath: /mnt
+      volumes:
+      - name: myvol
+        persistentVolumeClaim:
+          claimName: nginx-pvc
+
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: nginx-svc
+spec:
+  selector:
+    app: nginx
+  ports:
+  - port: 8080
+
+---
+apiVersion: route.openshift.io/v1
+kind: Route
+metadata:
+  name: nginx-route
+spec:
+  host: ""
+  path: /
+  to:
+    kind: Service
+    name: nginx-svc
+  tls:
+    insecureEdgeTerminationPolicy: Redirect
+    termination: edge
+
+---
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: nginx-pvc
+spec:
+  resources:
+    requests:
+      storage: 1Gi
+  accessModes:
+    - ReadWriteOnce
+  storageClassName: standard-rwo
+```
+
+The deployment is using a `PersistentVolumeClaim` for our example.  
+
+Now we have our running `nginx` pod, we want want to copy the content of the PVC to Allas. We will use a new deployment with a `rclone` Docker image.  
+
+Create a `rclone.conf` with your `access_key_id` and `secret_access_key`.  
```

Comment:
We should have a paragraph that clarifies that here is where the tutorial starts. Or move the code above to a separate section called "preparing a dummy deployment". This deployment can be used by the second method and we need to make it easier to navigate.

Reply:
"where the tutorial starts" meant "after the dummy deployment is created".

Reply:
I moved this section to the top and explained that for our tutorial we'll deploy a nginx server